### ISSUE TITLE
JAWS 2019 and Comboboxes with Grids and Listboxes have no label

### ISSUE THREAD

**Issue by ScreenReaderTester** [FreedomScientific/VFO-standards-support#171]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Go to https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/grid-combo.html OR https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html
> 2. Press Insert+F5 to "Select a Form Field"
> 3.  The list shows the combo as "Unlabeled"

### Expected result

> The comboboxes should be labeled per 
https://www.w3.org/TR/wai-aria-1.1/#combobox
and https://www.w3.org/TR/wai-aria-practices/#combobox

### Actual result

> "Select a Form Field" shows comboboxes as unlabeled

### Example

> A test case that demonstrates the issue - 
https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/grid-combo.html 
https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html

#### JAWS version and build number

JAWS 2019.1812.49 ILM
 
#### Operating System and version

Windows 7 Professional

#### Browser and version:

Firefox Quantum
Extended Support Release
60.5.0esr (32-bit)

**Comment by carmacleod** [comment]
I believe this is a duplicate of #77.

**Comment by scottaohara** [comment]
It is a duplicate.  Thank you
